### ISSUE TITLE
Update your binary on the forum?

### ISSUE THREAD

**Issue by melroy89** [BSG-75/RA3Bar-RA3Launcher#11]
Why not update your forum post with your latest binary as well??

https://www.gamereplays.org/community/index.php?s=&showtopic=1013524&view=findpost&p=10136316

**Comment by lanyizi** [comment]
Thanks for reminding me about that, I forgot about it xD
Will do it as soon as I have time.

**Comment by AlJohri** [comment]
Or maybe remove the binary from the forum and direct folks to Github [releases](https://github.com/lanyizi/RA3Bar-RA3Launcher/releases)?

Related:
- https://github.com/lanyizi/RA3Bar-RA3Launcher/pull/7
- https://github.com/lanyizi/RA3Bar-RA3Launcher/issues/5
- https://github.com/lanyizi/RA3Bar-RA3Launcher/releases/tag/v0.92

**Comment by melroy89** [comment]
Yes, even better. Since now it's still not up to date in the forum.
